build/vsts-ci.yml

```diff
@@ -182,9 +282,12 @@ phases:
 ################################################################################
   dependsOn:
   - Linux
+  - Linux_arm64
+  - Linux_arm64
```

Comment:
These 3 Linux entries look wrong.

1. Linux => Linux_x64
2. Linux_arm64 is duplicated
3. Linux_arm doesn't have an entry

Reply:
Good catch. I have fixed those.

Changed Linux to Linux_x64 everywhere.
One of the Linux_arm64 has been changed to just Linux_arm